packages/frontend/core/src/bootstrap/shared-worker.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This is a wrapper for SharedWorker,
+ * added the `name` parameter to the `SharedWorker` URL so that
+ * multiple `SharedWorkers` can share one script file.
+ */
+const rawSharedWorker = globalThis.SharedWorker;
+
+function PatchedSharedWorker(
```

Comment:
this is dangerous.
must be removed in the future.
there should be only one shared worker. avoid wasting too much computer resources.